src/components/CityCard/CityCard.jsx

```diff
@@ -0,0 +1,145 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useParams, useNavigate } from 'react-router-dom';
+import { fetchWeatherAction } from '../../redux/weather/weatherSlice';
+import back from '../../assets/images/back.png';
+import './style.css';
+
+function CityCard() {
+  const { id } = useParams();
+  const dispatch = useDispatch();
+  const history = useNavigate();
+
+  function handleGoBack(e) {
+    e.preventDefault();
+    history('/');
+  }
+
+  useEffect(() => {
+    dispatch(fetchWeatherAction(id));
+  }, []);
+
+  const state = useSelector((state) => state);
+  const { weather, loading, error } = state;
+
+  const sunrise = weather?.sys.sunrise;
+  const date1 = new Date(sunrise * 1000);
+  const hours1 = date1.getHours();
+  const minutes1 = `0${date1.getMinutes()}`;
+  // const seconds1 = `0${date1.getSeconds()}`;
+  // const formattedTime = `${hours1} : ${minutes1.substr(1, 2)} : ${seconds1.substr(1, 2)}`;
+  const formattedTime = `${hours1} : ${minutes1}`;
+
+  const sunset = weather?.sys.sunset;
+  const date2 = new Date(sunset * 1000);
+  const hours2 = date2.getHours();
+  const minutes2 = `0${date2.getMinutes()}`;
+  const formattedTime2 = `${hours2} : ${minutes2.substr(1, 2)}`;
+
+  return (
+    <div className="city-weather-details">
+      <section>
+        {loading ? (
+          <h1>Loading please wait...</h1>
+        ) : error ? (
+          <h1>
+            {error?.message}
+          </h1>
+        ) : (
+          <div className="weather-details">
+            <input className="go-back" type="image" src={back} onClick={handleGoBack} alt="go-back" />
+            <div className="the-city">
+              <h2 className="m-0">
+                {weather?.name}
+                {', '}
+                {weather?.sys?.country}
+              </h2>
+            </div>
+            <div className="sunrise">
+              <h2 className="m-0">Sunrise&nbsp;&nbsp;</h2>
+              <h2 className="m-0 f-12">
+                {formattedTime}
+                {' AM'}
+              </h2>
+            </div>
```

Comment:
- [ ] Kindly make the design of your Details page to be identical to that of the design template provided. Kindly the unique background feature used for the grid cells and stick to the layout for the details page.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/169412738-2c1b413f-35c9-4d14-8605-ab1bea8cbfd3.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/169412802-8cc51d4d-6821-4256-8078-b8f54e36b024.png)